torch/csrc/jit/codegen/cuda/lower2device.cpp

```diff
@@ -109,14 +109,30 @@ void GpuLower::lower() {
   // Compute thread predicates
   ThreadPredicateMap preds(fusion_);
 
-  // Compute root-domain mappings
-  ComputeAtRootDomainMap ca_root_map;
-  ca_root_map.build();
+  // In the future we may directly use this map, but for now it will propagate
+  // and validate (to some extent) the parallelization strategy.
+  // TODO: Does this have to go before the next mapping?
```

Comment:
Fix comment, no longer TODO, has to be done first since right now we're setting parallel strategy on fusion iter domains and need to do it before lowering.